react/components/RenderProvider.tsx

```diff
@@ -1108,26 +1132,17 @@ class RenderProvider extends Component<Props, RenderProviderState> {
       variables: Record<string, any>
     }>
   ) => {
-    forEach(({ data, query, variables }) => {
-      try {
-        this.apolloClient.writeQuery({
-          query: parse(query),
-          data: JSON.parse(data),
-          variables,
-        })
-      } catch (error) {
-        console.warn(
-          `Error writing query from render-server in Apollo's cache`,
-          error
-        )
-      }
-    }, queryData)
+    hydrateApolloCache(
```

Comment:
interesting change. has nothing to do with prefetch, has it? (no problem with it, just wondering)

Reply:
ah, i get it, after reading the code further